pyproject.toml

```diff
@@ -14,3 +14,8 @@ skip_fails = true
 # We keep the following for now because it is used to identify
 # and remove/edit old comments the bot posted in the past.
 pull_request_substring = "issues related to the changelog"
+
+[tool.astropy-bot.basebranch_checker]
+
+enabled = true
+basebranch = master
```

Comment:
Do I need quotes here? Not sure why the diff has red.

Reply:
Yes, I think you need the quotes, as the toml validator complains about the value without it:
```
Error on line 21:

Expected "+", "-", "[", "false", "true", "{", DoubleQuote, FullDate (YYYY-mm-dd), SingleQuote, ThreeDoubleQuotes, ThreeSingleQuotes, Whitespace, [0-9], or [1-9] but "m" found.
```

https://www.toml-lint.com/

Reply:
Thanks. Fixed!